index.js

```diff
@@ -35,37 +35,6 @@ const DESCRIPTION_MAP = {
     PENDING: 'Parked it as Pending...'
 };
 
-/**
- * Check the status code of the server's response.
- *
- * If there was an error encountered with the request, this will format a human-readable
- * error message.
- * @method checkResponseError
- * @param  {HTTPResponse}   response                               HTTP Response from `request` call
- * @param  {Number}         response.statusCode                    HTTP status code of the HTTP request
- * @param  {String}         [response.body.error.message]          Error message from the server
- * @param  {String}         [response.body.error.detail.required]  Error resolution message
- * @return {Promise}                                               Resolves when no error encountered.
- *                                                                 Rejects when status code is non-200
- */
-function checkResponseError(response, caller) {
```

Comment:
this has been moved to https://github.com/screwdriver-cd/request/blob/master/index.js